packages/tabs/src/Tabs.tsx

```diff
@@ -231,25 +231,31 @@ function Tabs({
     }
 
     const isTabSelected = index === selected;
-    const { disabled, onClick } = child.props;
+    const { disabled, onClick, onKeyDown, className, ...rest } = child.props;
```

Comment:
```suggestion
    const { onClick, onKeyDown, className, ...rest } = child.props;
```

Reply:
We use the disabled value a couple of times (in className concat and onClick)

Reply:
If you feel strongly though, happy to make these changes

Reply:
no worries